### DISCUSSION TITLE
Community

### DISCUSSION THREAD

**Discussion by byteduck** [byteduck/duckOS#44] [Announcements]
Hello everyone!

I recently made a Discord server for duckOS, in the hopes that those who wish to contribute would have a place to communicate. Upon further consideration, however, I figured I kind of jumped the gun. I know a lot of other FOSS projects use Discord, but many others prefer more open solutions like IRC or Matrix.

For those who care to answer, what would you like to see in terms of a community around duckOS?

**Comment by MarcoCicognani** [comment]
Hi Aaron,
What do you think of Telegram group?

**Reply by byteduck** [reply]
I've heard that suggested before, I'll look into it to see if it's a viable solution. Thanks!
